example/scp.c

```diff
@@ -145,6 +145,10 @@ int main(int argc, char *argv[])
         goto shutdown;
     }
 
+#ifdef WIN32
```

Comment:
It's not strictly necessary for support of > 2gb files but it is a general win32 thing since write() on win32 will always translate \n -> \n\r which makes binary transfers unusable.  Let me close this pr and I'll send you one with just the off_t change.  I'll send it against upstream once the 2gb fixes are in.